.linelint.yml

```diff
@@ -0,0 +1,12 @@
+rules:
+  # checks if file ends in a newline character
+  end-of-file:
+    # set to true to enable this rule
+    enable: true
+
+    # set to true to disable autofix (if enabled globally)
+    disable-autofix: false
```

Comment:
Is there any point to have this enabled here?

Reply:
This one replaces the one in `py.test -vs tests/integration/format_tests.py`. If we don't wan't that at all, we can remove everything.

Reply:
I was referring to the `disable-autofix`. Not sure we're talking about the same.

Reply:
Yeah, now I got it. Fixing